Automattic/jetpack
# Merge trunk to paid-newsletters WIP

p1667856843675049-slack-C01B6KEJ5GE

The paid-newsletters login/checkout functionality wasn't working for me today because `jetpack-require-lib` has undergone some changes.  This PR pulls the latest trunk functionality to get around this difficulty.

## Testing instructions

1. Using the origin/paid-newsletters branch attempt to subscribe or login to a subscribed newsletter site.  There will be no error but things will not function as expected.
2. Use this PR to do the same thing, and validate that it's now working.
